### PR TITLE
#965 Added team name to project page

### PR DESCRIPTION
**Pull request by AShyamkumar19**
## Changes
Added team name to the project page. 

_Explanation of changes goes here_
There are multiple teams in the Finishline database that created projects. So every project is tagged with the creator of the project which is the team. It's under the Project details page under the overview tab. 

## Screenshots

_If you made UI changes you must post a screenshot of the whole page for each change in 1) a normal sized window and 2) the smallest possible window_

_If you did any manual testing (e.g., with Postman), put screenshots of the http request and before and after of the db_

_If none of this applies, you can delete this section_

WIth Name

none

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes # (#965 )

### CODE REVIEW

**Review by zmar234 — CHANGES_REQUESTED** [review]
Hey Aarav, projects can have multiple teams so the way it currently is can get really messy with multiple teams. Due to this, we think the teams should go into the details section. I think the best way to go about this is to have a **Teams:** like the other data points with all the teams listed under the Details header but above the grid with all the data. (It'll go where the blue arrow is)

270138249-c4934db1-2ca2-4add-84b0-10d2b98d12dd

File: src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx [hunk 1]
```diff
@@ -160,7 +160,14 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ project, en
     </Box>
   );
 
-  const pageTitle = `${wbsPipe(project.wbsNum)} - ${project.name}`;
+  let teamName = '';
+  try {
+    teamName = project.teams[0].teamName;
```
Comment (walker-sean) [inline src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx:165]: Make sure you display all teams, not just the first one.

Comment (walker-sean) [reply src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx:165]: Take a look at some examples in the codebase of how to construct a list from an array separated by commas

**Review by walker-sean — CHANGES_REQUESTED** [review]
Nice job so far! 
Your prettier (style) check is failing, so make sure to fix that for your next commit. This is usually fixed by doing ctrl/cmd + S if you have the Prettier extension on VSCode. 
As for your failing test, this is because the test is looking for a specific title, which you have changed, so this is a case where you need to update the test.

**Review by zmar234 — DISMISSED** [review]
Looks good to me design-wise, great job!

File: src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectDetails.tsx [hunk 2]
```diff
@@ -33,31 +35,30 @@ const ProjectDetails: React.FC<ProjectDetailsProps> = ({ project }) => {
         </Typography>
 
         <Grid container spacing={2}>
+          <Grid item display="flex" alignItems="center" xs={12} sm={6}>
+            <GroupIcon sx={{ mr: 2 }} />
+            <DetailDisplay label="Teams" content={getProjectTeamsName(project)} paddingRight={1} />
```
Comment (walker-sean) [inline src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectDetails.tsx:40]: Change the label to `"Team"` when the project has exactly one team

**Review by walker-sean — CHANGES_REQUESTED** [review]
Nice work!

File: src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectDetails.tsx [hunk 3]
```diff
@@ -33,31 +35,34 @@ const ProjectDetails: React.FC<ProjectDetailsProps> = ({ project }) => {
         </Typography>
 
         <Grid container spacing={2}>
+          <Grid item display="flex" alignItems="center" xs={12} sm={6}>
+            <GroupIcon sx={{ mr: 2 }} />
+            <DetailDisplay
+              label={getProjectTeamsName(project).split(', ').length > 1 ? 'Teams' : 'Team'}
```
Comment (walker-sean) [inline src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectDetails.tsx:41]: Use `project.teams.length` to get the number of teams

**Review by walker-sean — CHANGES_REQUESTED** [review]
Looks good, just one more thing

**Review by walker-sean — APPROVED** [review]
LGTM!

**Pull request merged**